geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/audit/AccumuloEventWriter.scala

```diff
@@ -79,8 +78,7 @@ class AccumuloEventWriter(connector: Connector, table: String) extends Runnable
 
 object AccumuloEventWriter {
 
-  val WriteInterval = SystemProperty("geomesa.accumulo.audit.interval", "5 seconds")
+  val WriteInterval: SystemProperty = SystemProperty("geomesa.accumulo.audit.interval", "5 seconds")
 
-  private val executor = MoreExecutors.getExitingScheduledExecutorService(new ScheduledThreadPoolExecutor(5))
-  sys.addShutdownHook(executor.shutdownNow())
+  private val executor = ExitingExecutor(new ScheduledThreadPoolExecutor(5), force = true)
```

Comment:
Nice refactoring.  Is there elsewhere this should be done?

Reply:
yeah, i replaced all the usages of guava's MoreExecutors with this